src/framework/shortcuts/internal/platform/macos/macosshortcutsinstancemodel.mm

```diff
@@ -26,13 +26,65 @@
 #include <QApplication>
 #include <QKeyEvent>
 
-#include <private/qguiapplication_p.h>
-#include <qpa/qplatformintegration.h>
-
 #include "log.h"
 
 using namespace muse::shortcuts;
 
+static std::map<UInt32, QString> specialKeysMap = {
+    { kVK_F1, "F1" },
+    { kVK_F2, "F2" },
+    { kVK_F3, "F3" },
+    { kVK_F4, "F4" },
+    { kVK_F5, "F5" },
+    { kVK_F6, "F6" },
+    { kVK_F7, "F7" },
+    { kVK_F8, "F8" },
+    { kVK_F9, "F9" },
+    { kVK_F10, "F10" },
+    { kVK_F11, "F11" },
+    { kVK_F12, "F12" },
+    { kVK_F13, "F13" },
+    { kVK_F14, "F14" },
+    { kVK_F15, "F15" },
+    { kVK_F16, "F16" },
+    { kVK_F17, "F17" },
+    { kVK_F18, "F18" },
+    { kVK_F19, "F19" },
+    { kVK_Space, "Space" },
+    { kVK_Escape, "Esc" },
+    { kVK_Delete, "Delete" },
+    { kVK_ForwardDelete, "Backspace" },
```

Comment:
These two should be swapped (see e.g. https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values#code_values_on_mac)
```suggestion
    { kVK_Delete, "Backspace" },
    { kVK_ForwardDelete, "Delete" },
```
(Yes, it's weird, but Apple has this weird habit of calling the backspace key "Delete". You can see it when you look up a picture of a MacBook with American keyboard layout.)